src/presence_analyzer/templates/presence_weekday.html

```diff
@@ -26,6 +26,7 @@
                     if(selectedUser) {
                         loading.show();
                         $chartDiv.hide();
+                        document.getElementById("profile_pic").src="https://intranet.stxnext.pl/api/images/users/" + selectedUser;
```

Comment:
You should build this url on backend. Thats why there is config header like port etc